content/en/docs/concepts/security/linux-kernel-security-constraints.md

```diff
@@ -0,0 +1,270 @@
+configurations at scale might be challenging, especially if you use all three
+features together. To help you to manage these configurations at scale, use a
+tool like the
+[Kubernetes Security Profiles Operator](https://github.com/kubernetes-sigs/security-profiles-operator).
+
+## Kernel-level security features and privileged containers {#kernel-security-features-privileged-containers}
+
+Kubernetes lets you specify that some trusted containers can run in *Privileged*
+mode. Any container in a pod can run in privileged mode to use operating system
+administrative capabilities that would otherwise be inaccessible. This is
+available for both Windows and Linux.
+
+Privileged containers explicitly override some of the Linux kernel constraints
+that you might use in your workloads, as follows:
+
+* **seccomp**: Privileged containers run as the `Unconfined` seccomp profile,
+  overriding any seccomp profile that you specified in your manifest.
+* **AppArmor**: Privileged containers ignore any applied AppArmor profiles. TODO:Get confirmation
+* **SELinux**: TODO
+
+### Privileged containers {#privileged-containers}
+
+<!-- Content from https://kubernetes.io/docs/concepts/workloads/pods/#privileged-mode-for-containers  -->
+
+Any container in a Pod can enable *Privileged mode* if you set the
+`privileged: true` field in the container
+[securityContext](/docs/tasks/configure-pod-container/security-context/).
+Privileged containers override or undo many other hardening settings such as the
+applied seccomp profile, AppArmor profile, or SELinux constraints. Privileged
+containers are given all Linux capabilities, including capabilities that they
+don't require. For example, a root user in a privileged container might be able
+to use the `CAP_SYS_ADMIN` and `CAP_NET_ADMIN` capabilities on the node,
+bypassing the runtime seccomp configuration and other restrictions.
+
+In most cases, you should avoid using privileged containers, and instead grant
+the specific capabilities required by your container using the `capabilities`
+field in the `securityContext`. Only use privileged mode if you have a
+capability that you can't grant with the securityContext. This is useful for
+containers that want to use operating system administrative capabilities such as
+manipulating the network stack or accessing hardware devices.
+
+### Windows privileged containers {#windows-privileged-containers}
+
+{{< feature-state for_k8s_version="1.26" state="stable" >}}
+
+In Windows, you can create a
+[Windows HostProcess pod](/docs/tasks/configure-pod-container/create-hostprocess-pod)
+by setting the `windowsOptions.hostProcess` flag on the security context of the
+pod spec. All containers in these pods must run as Windows HostProcess
+containers. HostProcess pods run directly on the host and can also be used to
+perform administrative tasks as is done with Linux privileged containers. In
+order to use this feature, the WindowsHostProcessContainers feature gate must be
+enabled.
```

Comment:
This windows privileged containers is better placed in a more generic page, as its implementation does not relate to Linux kernel security constraints.

We are probably better off pointing folks interested on `HostProcess` to https://kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod/